pkg/chunk/gcp/table_client.go

```diff
@@ -81,6 +83,10 @@ func alreadyExistsError(err error) bool {
 	return ok && strings.Contains(serr.Message(), "already exists")
 }
 
+func familyAlreadyExistsError(err error) bool {
+	return strings.Contains(err.Error(), "AlreadyExists")
```

Comment:
Is there a better way to detect this?  Like `alreadyExistsError` above - can we reuse the same code?